app/models/spree/variant.rb

```diff
@@ -199,6 +199,11 @@ def amount_in(currency)
       price_in(currency).try(:amount)
     end
 
+    def changed?
+      # Changes to price are saved after_save
```

Comment:
shouldnt it be a question? It sounds as statement. Its confusing. Is the method setter or getter?  Or comment does not describe this method and is related to overall workflow of persisting `Variant`s data to DB?

Reply:
`changed?` is rails method telling us if any attributes on the model have been modified : https://api.rubyonrails.org/classes/ActiveModel/Dirty.html#method-i-changed-3F. 
And also, method ending with a `?` a ruby code convention indicating the method returns a boolean.

Reply:
I can see how the comment is confusing: it doesn't describe what this method does, it's explaining why the method is here. (it's overriding the rails method that Gaetan referred to, thanks).
I'll update the comment to be a bit clearer.